addons/survey/models/survey_survey.py

```diff
@@ -71,11 +71,11 @@ def _get_default_access_token(self):
     users_can_signup = fields.Boolean('Users can signup', compute='_compute_users_can_signup')
     public_url = fields.Char("Public link", compute="_compute_survey_url")
     # statistics
-    invite_count = fields.Integer("Invite", compute="_compute_survey_statistic")
-    answer_count = fields.Integer("Started", compute="_compute_survey_statistic")
-    answer_done_count = fields.Integer("Completed", compute="_compute_survey_statistic")
-    certified_count = fields.Integer("Certified", compute="_compute_certified_count")
-
+    answer_count = fields.Integer("Registered", compute="_compute_survey_statistic")
+    answer_done_count = fields.Integer("Attempts", compute="_compute_survey_statistic")
+    answer_score_avg = fields.Float("Avg Score %", compute="_compute_survey_statistic")
+    certified_count = fields.Integer("Certified", compute="_compute_survey_statistic")
+    certified_ratio = fields.Integer("Certified Ratio", compute="_compute_certified_ratio")
```

Comment:
As certified ratio is also used to compute the success ratio (when survey is not a certification), I think we should call this success_ratio